specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -1996,16 +2013,16 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DesktopVirtualization/hostPools/{hostPoolName}/updateValidationOperationResults/default": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DesktopVirtualization/locations/{location}/operationResults/{operationResultId}": {
```

Comment:
refer to https://github.com/Microsoft/api-guidelines/blob/vNext/Guidelines.md#1324-operations-resource, should we use /operations instead of /operationResults?

Reply:
No /operations is used to enumerate all the actions exposed by your RP. See [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/proxy-api-reference.md#exposing-available-operations](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/proxy-api-reference.md#exposing-available-operations)